src/PokemonGymImpl.java

```diff
@@ -184,7 +184,7 @@ public void performAttackPlayer(Pokemon pokemon, Pokemon gymPokemon, String atta
                 electric = new ElectricPokemon(pokemon.getName(), pokemon.getLevel(), pokemon.getHp(), pokemon.getFood(), pokemon.getSound());
                 switch (choosenAttack) {
                     case "thunderpunch" -> electric.thunderPunch(pokemon, gymPokemon);
-                    case "electroball" -> electric.electroBall(pokemon, gymPokemon);
+                    case "electroball" -> electric.electricBall(pokemon, gymPokemon);
```

Comment:
Ik denk dat je beter de naam van je aanval methode in je ElectricPokemon class kunt aanpassen, in plaats van hier.